ScoutHelper/Windows/ImGuiPlus.cs

```diff
@@ -48,6 +48,8 @@ public static class ImGuiPlus {
 		return finalCursorPos;
 	}
 
+	public static float ScaledFontSize() => 4 * MathF.Pow(ImGui.GetFontSize(), 0.6f);
```

Comment:
i'm tired of font scaling not actually being entirely predictable. after experimentation, i found that an exponent of about 0.6 makes it mostly predictably scale, but i need to experiment with this more before trying to apply it in more places. right now it's only used in the instance inputs.